packages/ui5-task-zipper/README.md

```diff
@@ -30,7 +30,7 @@ Default value: `<app-id.zip>`
 List of files to be included in the ZIP archive relative to the project root or Map of of files to be included in the ZIP archive relative to the project root and target path in the ZIP archive.
 
 - onlyZip: `true|false`
-Set this to `true` if you also want to generate the unzipped resources in the `dist` folder. Otherwise, it will only create the zipped archive.
+Set this to `false` if you also want to generate the unzipped resources in the `dist` folder. Otherwise, it will only create the zipped archive.
```

Comment:
I would even phrase this completely different:

Set this to `true` to omit the resources contained in the zip from the build result (typically in the `dist` folder). By default, the build result contains all resources and the zip.